tests/integration-tests/tests/schedulers/test_slurm_accounting.py

```diff
@@ -0,0 +1,308 @@
+            capabilities=["CAPABILITY_AUTO_EXPAND"],
+        )
+    cfn_stacks_factory.create_stack(database_stack)
+    logging.info("Creation of stack %s complete", database_stack_name)
+
+    return database_stack
+
+
+@pytest.fixture(scope="module")
+def database_factory(request, cfn_stacks_factory, vpc_stacks):
+    created_database_stacks = defaultdict(dict)
+
+    logging.info("Setting up database_factory fixture")
+
+    def _database_factory(
+        existing_database_stack_name,
+        test_resources_dir,
+        region,
+    ):
+        if existing_database_stack_name:
+            logging.info("Using pre-existing database stack named %s", existing_database_stack_name)
+            return existing_database_stack_name
+
+        if not created_database_stacks.get(region, {}).get("default"):
+            logging.info("Creating default database stack")
+            database_stack = _create_database_stack(
+                cfn_stacks_factory, request, test_resources_dir, region, vpc_stacks[region]
+            )
+            created_database_stacks[region]["default"] = database_stack.name
+
+        logging.info("Using database stack %s", created_database_stacks.get(region, {}).get("default"))
+        return created_database_stacks.get(region, {}).get("default")
+
+    yield _database_factory
+
+    for region, stack_dict in created_database_stacks.items():
+        stack_name = stack_dict["default"]
+        if request.config.getoption("no_delete"):
+            logging.info(
+                "Not deleting database stack %s in region %s because --no-delete option was specified",
+                stack_name,
+                region,
+            )
+        else:
+            logging.info(
+                "Deleting database stack %s in region %s",
+                stack_name,
+                region,
+            )
+            cfn_stacks_factory.delete_stack(stack_name, region)
+
+
+@pytest.fixture(scope="function")
+def test_resources_dir(datadir):
+    return datadir / "resources"
+
+
+def _get_slurm_database_config_parameters(database_stack_outputs):
+    return {
+        "database_host": database_stack_outputs.get("DatabaseHost"),
+        "database_admin_user": database_stack_outputs.get("DatabaseAdminUser"),
+        "database_secret_arn": database_stack_outputs.get("DatabaseSecretArn"),
+        "database_client_security_group": database_stack_outputs.get("DatabaseClientSecurityGroup"),
+    }
+
+
+def _get_expected_users(remote_command_executor, test_resources_dir):
+    users = remote_command_executor.run_remote_script(str(test_resources_dir / "get_accounting_users.sh")).stdout
+    for user in users.splitlines():
+        logging.info("  Expected User: %s", user)
+    return users.splitlines()
+
+
+def _get_expected_jobs(remote_command_executor, test_resources_dir):
+    results = list(
+        remote_command_executor.run_remote_script(str(test_resources_dir / "run_jobs.sh")).stdout.splitlines()
```

Comment:
Rather than a bash script, I would use the methods provided by the `slurm_commands` class (for example, check [here](https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/schedulers/test_slurm.py#L1646-L1653)).

Reply:
Functionality moved to SlurmCommands class.